README.md

```diff
@@ -1,82 +1,65 @@
-  - If it's legal Spox, it (should be) legal ONNX!
-
-The main goal of Spox is to provide a robust and Pythonic framework for developing libraries building ONNX graphs, such as converters or other custom applications.
+Converting a trained model into ONNX entails replicating its inference logic with ONNX operators.
+In the past that constituted a major undertaking.
+Based on those experiences we designed Spox from the ground up to make the process of writing converters (and ONNX models in general) as easy as possible.
 
 ## Installation
 
-Spox is published on conda-forge and can be installed as expected:
+With pip:
 
 ```bash
-conda install spox
+pip install spox
 ```
 
-## Getting started
-
-### Constructing
-
-In Spox, most of the time you'll be working with `Var` objects - **variables**. You either create them yourself as arguments for a model, or receive them from another source, for example when you're writing a converter in a library.
-
-You may print out the `Var` or check its `Var.type` to learn a bit more about it.
-
-To perform operations on a `Var`, use an _opset_ (operator set) module like `spox.opset.ai.onnx.v17` or `spox.opset.ai.onnx.ml.v3` - which correspond to `ai.onnx@17` and `ai.onnx.ml@3`, the standard opsets. These are pre-generated for you and you may import them as required.
-
-For instance, using the default opset you could write a function that given two variables returns their [geometric mean](https://en.wikipedia.org/wiki/Geometric_mean):
-
-```python
-from spox import Var
-from spox.opset.ai.onnx import v17 as op
+Spox is also available on conda-forge and can be installed as expected:
 
-def geometric_mean(x: Var, y: Var) -> Var:
-    return op.sqrt(
-      op.mul(x, y)
-    )
+```bash
+conda install spox
 ```
 
-Since ONNX is tensor-oriented, this code assumes that `x` and `y` are floating point tensors with matching (broadcastable) shapes, and the geometric mean is computed elementwise. You may learn more about what types and shapes are acceptable for given operators by checking their docstring.
+## Quick start
 
-Performing operations on Variables creates further Variables, all of which keep track of what computation has been performed so far. If an operation is determined to be illegal (for example due to mismatching types/shapes), an exception will be immediately raised by Spox.
+Spox users interact most commonly with `Var` objects - **variables** which will have values assigned to them at runtime.
+The initial `Var` objects which represent the *argument*s of a model (the model inputs in ONNX nomenclature) are created with an explicit type and shape using the `spox.argument` function.
+All further `Var` objects are created by calling functions which take existing `Var` objects as inputs and produce new `Var` objects as outputs.
+Spox provides such functions for all operators in the standard grouped by domain and version inside the `spox.opset` module.
```

Comment:
```suggestion
Spox provides such functions - _operator constructors_ - for all operators in the standard grouped by domain and version inside the `spox.opset` module.
```

Reply:
My thought here: It does not matter to the user what is happening inside the functions. Why introduce the term "operator constructor" at all?

Reply:
Most documentation articles use that term, including the listings. I think it's just convenient to call these things somehow, as they have an important role.

Reply:
But from the user's point of view it makes no difference if you are dealing with your own functions or with a "constructor function" from spox. In any case, I don't think we need to introduce the name here in the readme.